rules/windows/process_creation/proc_creation_win_iis_service_account_password_dumped.yml

```diff
@@ -0,0 +1,27 @@
+title: Microsoft IIS Service Account Password Dumped
+id: 2d3cdeec-c0db-45b4-aa86-082f7eb75701
+status: experimental
+description: Detects the Internet Information Services (IIS) command-line tool, AppCmd, being used to list passwords
+author: Tim Rauch
+references:
+    - https://www.elastic.co/guide/en/security/current/microsoft-iis-service-account-password-dumped.html
+date: 2022/09/28
+logsource:
+    category: process_creation
+    product: windows
+tags:
+    - attack.credential_access
+    - attack.t1003
+detection:
+    selection_name:
+        - Image|endswith: '\appcmd.exe'
+        - OriginalFilename: 'appcmd.exe'
+    selection_args:
+        CommandLine|contains|all:
+            - '/list'
+            - '/text'
+            - 'password'
```

Comment:
- Please add spaces before the arguments
- Also check if the "-" (dash) can be used instead of a slash (If it's the case please add those variations)